css assignment/aboutme.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
make contetnt to center of screen add some padding margin and width

Reply:
Done